system_tests/utils.go

```diff
@@ -262,7 +262,7 @@ func rabbitmqAMQPSPublishToNewQueue(message, username, password, hostname, amqps
 	return nil
 }
 
-func rabbitmqAMQPSGetMessageFromQueue(username, password, hostname, amqpsPort, caFilePath string) (string, error) {
+func getMessageFromQueueAMQPS(username, password, hostname, amqpsPort, caFilePath string) (string, error) {
```

Comment:
If we change the publish function to as suggested, change this to be consistent `getMessageFromQueueViaAMQPS`

Reply:
I like `getMessageFromQueueAMQPS ` since it's shorter 😆